overlays/development/wsl2/mariadb_pv.yaml

```diff
@@ -30,5 +30,5 @@ spec:
   capacity:
     storage: 1Gi
   hostPath:
-    path: /run/desktop/mnt/host/c/kubernetes/idc/mariadb-files
```

Comment:
This change only changes the bound path to the backend OS so this means you changed the name of the directory in the tar file?  Or the name was always the same and I had it wrong in whatever I was using?  I don't think there is a conflict here, I am just making sure.

Reply:
Yes, as of https://github.com/jhu-idc/idc-isle-dc/pull/53 the snapshot tar file has a directory called `mariadb-dump` in it containing sql dumps, but _not_ `mariadb-files`. 

basically, the snapshot has this in it:
```
drupal
mariadb-dump
solr
```

After the stack starts up, you'll see
```
activemq-data 
cantaloupe-data 
drupal 
mariadb-data
mariadb-dump
solr
``` 
`mariadb-data` solely fulfills the mariadb container's persistent volume claim (that's where mariadb keeps is raw database files), but isn't part of the snapshot any more.